Technical-Steering-Committee/coordinated-disclosure-details.md

```diff
@@ -0,0 +1,45 @@
+# What does the Zowe Disclosure process look like?
+
+There are two main processes to obtain information about security vulnerabilities. One process is used by
+external stakeholders, and the other process is used by internal stakeholders. 
+
+## Step 1: New issue reported
+
+An external stakeholder reports issues via the zowe-security@openmainframeproject.org mailing list. 
+This issue is then discussed at the weekly Security Workgroup call. At this time, issues within
```

Comment:
One minor concern, if the vulnerability is high/critical - we probably won't wait until the weekly call - we would probably call a meeting asap, right?  So perhaps reword as follows:  "This issue is then discussed by the Security Workgroup.  As preparation for the discussion, issues within the ...."

Reply:
I added latest, as the truth is that this is the latest moment, when the issue will be discussed.